project/plugins.sbt

```diff
@@ -4,5 +4,6 @@ addSbtPlugin("com.geirsson"      % "sbt-scalafmt" % "1.4.0")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-git"      % "0.9.3")
 addSbtPlugin("de.heikoseeberger" % "sbt-header"   % "4.1.0")
 addSbtPlugin("org.foundweekends" % "sbt-bintray"  % "0.5.3")
+addSbtPlugin("com.timushev.sbt"  % "sbt-updates"  % "0.3.4")
```

Comment:
Please remove this line, I use it solely as a global plugin (until there will be CI automated releases, if that will ever happen).

Reply:
Done.